model/service.go

```diff
@@ -54,6 +54,9 @@ type Service struct {
 	// service DNS name.  External services are name-based solution to represent
 	// external service instances as a service inside the cluster.
 	ExternalName string `json:"external"`
+
+	// ServiceAccounts specifies the service accounts that run the service.
+	ServiceAccounts []string `json:"serviceaccounts,omitempty"`
```

Comment:
Why is this back?
@rshriram why do you want this to be a first class element but not have service instance accounts?